### ISSUE TITLE
Search Results not able to edit column value after adding selected property

### ISSUE THREAD

**Issue by CasmahaOGD** [microsoft-search/pnp-modern-search#3910]
**Version used**
4.12.2

**Describe the bug**
After adding a RefinableString to the Selected Properties of a Search Results webpart I am unable to edit any column value except for handlebars.
Clicking on the column value selects the currently selected value but I am unable to type anything else, clicking the drop down arrow also doesn't do anything.
Trying to add a new column also will not let me type or select any value in the column value box.
Already existing Search Results web parts are editable until I add a new Selected Property.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Go to a Search Results webpart
3. Click on Selected Properties and select a new one or type a RefinableString to add it.
4. Save and go to Manage Columns
5. Try to edit any column value or add a new column
6. See error

**Expected behavior**
Expected behavior is to be able to edit the column values at any time

**Screenshots**
Here the column value is selected and the dropdown menu should be showing so another value can be selected.
![image](https://github.com/user-attachments/assets/7c1edf28-08f5-4c95-a810-768f841c25a4)

**Desktop (please complete the following information):**
Chrome

**Comment by kasperbolarsen** [comment]
You haven't reset the Query Template by any chance? It is precondition that we have an applied query template ( on page 1)

**Comment by CasmahaOGD** [comment]
Nope I haven't, I also get the same result in 2 different tenants, also tried reinstalling PnP Modern search entirely and still get the same result.

**Comment by kasperbolarsen** [comment]
I can only get to the state in your screenshot when I have an invalid search query. I guess your current query returns something.
Can you share the current setup by exporting the settings?

**Comment by CasmahaOGD** [comment]
[webpartproperties.json](https://github.com/user-attachments/files/16675958/webpartproperties.json)
There you go.
As you have guessed I do get results in the search result web part but am unable to edit the column values.

**Comment by kasperbolarsen** [comment]
I can't reproduce this issue. Is it only an issue on a specific site collection or tenant?

**Comment by CasmahaOGD** [comment]
I have this issue within 2 different tenants, on different site collections as well.
And when I create a new site and try the same I get the same issue (in both tenants), without changing any other configuration (other than connecting the search box).

**Comment by CasmahaOGD** [comment]
Yes so I also tried that, everything works fine in the new web part until I add a RefinableString by typing it in the Selected Properties. After doing this and trying to add a new column the same issue comes up (in both tenants).

Now I have tried downgrading the version of PnP Modern Search and on 4.11.0.0 I don't have this issue.
